File: myheroes.api/myHeroesAPI/HeroModel.cs
namespace myHeroesAPI
{
    public class HeroModel
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string NameHero { get; set; } = string.Empty;
        public List<string>? Superpowers { get; set; }=null;
        public DateTime Birthdate { get; set; }
        public double Height { get; set; } 
        public double Weight { get; set; }
    }
}

File: myheroes.web/src/app/models/hero.model.ts
export class Hero {
    id: number;
    name: string;
    nameHero: string;
    superpowers: SupePower[];
    birthdate?: Date;
    height: number;
    weight: number;
}

export class SupePower {
    id:number;
    superPower:string;
    description:string;
}